Content.Client/UserInterface/Systems/Guidebook/GuidebookUIController.cs

```diff
@@ -163,12 +147,14 @@ private void OnWindowOpen()
     /// <param name="includeChildren">Whether or not to automatically include child entries. If false, this will ONLY
     /// show the specified entries</param>
     /// <param name="selected">The guide whose contents should be displayed when the guidebook is opened</param>
+    /// <param name="closeIfShown">Whether or not to close the guidbook window if it's already opened</param>
     public void OpenGuidebook(
         Dictionary<ProtoId<GuideEntryPrototype>, GuideEntry>? guides = null,
         List<ProtoId<GuideEntryPrototype>>? rootEntries = null,
         ProtoId<GuideEntryPrototype>? forceRoot = null,
         bool includeChildren = true,
-        ProtoId<GuideEntryPrototype>? selected = null)
+        ProtoId<GuideEntryPrototype>? selected = null,
+        bool closeIfShown = true)
```

Comment:
Much better API design to keep this on a separate method instead of making toggle functionality something done via passing args.